src/contracts/pods/EigenPod.sol

```diff
@@ -99,17 +99,17 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     event EigenPodStaked(bytes pubkey);
 
     /// @notice Emitted when an ETH validator's withdrawal credentials are successfully verified to be pointed to this eigenPod
-    event ValidatorRestaked(uint40 validatorIndex);
+    event ValidatorRestaked(uint40 validatorIndex, bytes32 validatorPubkeyHash, bytes validatorPubkey);
```

Comment:
to my knowledge, events with dynamic length arguments like `bytes validatorPubkey` will _actually_ output the keccak256 hash of the argument. might be fine for indexing, but if `validatorPubkeyHash` is already the keccak256 hash (and not a different kind of hash) then this is might be redundant info to be emitting?
will dig in more, this is a bit a note to myself.

Reply:
Oh interesting, didn't know that.  Its really important for the indexing that they have both the hash and the preimage.

Reply:
gotcha. I think because there's some extra padding happening here before hashing, I can see how having both in the event makes sense.

Reply:
yup, exactly

Reply:
cool, resolving thread, thanks for going on this journey with me.